scripts/crew/CrewMember.gd

```diff
@@ -161,21 +170,35 @@ func _on_walking_state_physics_processing(_delta: float) -> void:
 		#set_movement_target(Vector2(x,y))
 
 func _on_working_state_entered() -> void:
-	print("working...")
+	print(data.name, ": working...")
 	state = STATE.WORK
 
 func _on_working_state_physics_processing(_delta: float) -> void:
 	randomise_target_position_in_room()
 
 func _on_working_state_exited() -> void:
-	print("stopped working")
+	print(data.name, ": stopped working")
 
 func can_assign() -> bool:
 	return Global.station.rooms.size() > 0
 
 func assign(room: Room, center: Vector2) -> void:
```

Comment:
Nice, I think we can modify this for furniture objects without too much trouble.